src/config/dark-sites.config

```diff
@@ -62,6 +62,7 @@ avengedsevenfold.com
 azeria-labs.com
 azm.azerialabs.com
 badboybill.com
+based.cooking
```

Comment:
![image](https://user-images.githubusercontent.com/25481501/125341999-90bd1180-e354-11eb-9023-97f63788d569.png)

Doesn't seem that dark to me.

Reply:
Could've sworn there was a dark theme at one time [This issue on the website's repo says so](https://github.com/LukeSmithxyz/based.cooking/issues/283) but it doesn't matter